packages/twenty-ui/src/display/icon/components/IconIllustrationOneToMany.tsx

```diff
@@ -0,0 +1,23 @@
+import { useTheme } from '@emotion/react';
+
+import IconIllustrationOneToManyRaw from '@ui/display/icon/assets/illustration-one-to-many.svg?react';
+import { IconComponentProps } from '@ui/display/icon/types/IconComponent';
+
+type IconIllustrationOneToManyProps = Pick<IconComponentProps, 'size'>;
+
+export const IconIllustrationOneToMany = (
+  props: IconIllustrationOneToManyProps,
+) => {
+  const theme = useTheme();
+  const size = props.size ?? theme.icon.size.lg;
+  const illustrationColors = theme.iconIllustration;
```

Comment:
const { color, fill } = theme.iconIllustration

I actually think that correct english would be illustrationIcon instead of iconIllustration change to be made everywhere :p)